src/applications/mhv-landing-page/components/LandingPageAlerts.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import IdentityNotVerified from '~/platform/user/authorization/components/IdentityNotVerified';
+import UnregisteredAlert from './UnregisteredAlert';
+import MhvRegistrationAlert from './MhvRegistrationAlert';
+import MhvBasicAccountAlert from './MhvBasicAccountAlert';
+import VerifyAndRegisterAlert from './VerifyAndRegisterAlert';
+
+const LandingPageAlerts = ({
+  signInService,
+  showsVerifyAndRegisterAlert,
+  unVerifiedHeadline,
+  userHasMhvAccount,
+  userHasMhvBasicAccount,
+  userRegistered,
+  userVerified,
+}) => {
+  if (userHasMhvBasicAccount && !userVerified) {
+    return <MhvBasicAccountAlert />;
+  }
+
+  if (
+    (signInService === 'idme' || signInService === 'logingov') &&
+    !userVerified &&
+    showsVerifyAndRegisterAlert
```

Comment:
Doesn't `showsVerifyAndRegisterAlert` also check the sign in service is idme or logingov and is LOA3? I don't think we should need the first two conditional checks:

```suggestion
    showsVerifyAndRegisterAlert
```